pyproject.toml

```diff
@@ -1,10 +1,85 @@
 [build-system]
-    build-backend = "setuptools.build_meta"
-    requires = [
-        "setuptools>=47",
-        "wheel>=0.34",
-        "cython>=3.0.8; python_implementation == 'CPython'", # Skip cython when using pypy
-    ]
+build-backend = "setuptools.build_meta"
+requires = [
+    "setuptools>=47",
+    "wheel>=0.34",
```

Comment:
Good point, I don't know why it is here tbh. I can test which setuptools version no longer needs it, and specify it.

Reply:
the seem to have removed that from the quickstart in version 61 https://github.com/pypa/setuptools/blob/v61.0.0/docs/userguide/quickstart.rst

Reply:
Thanks for bisecting this!

Reply:
61 requires >= 3.7, so it should be fine to specify. Moreover, this spec is oftentimes more or less moot, since PEP 517 stuff is normally performed in a fresh env.